jobs/JGDAS_ENKF_DIAG

```diff
@@ -22,31 +22,33 @@ GDATE=$(${NDATE} -${assim_freq} ${CDATE})
 gPDY=$(echo ${GDATE} | cut -c1-8)
 gcyc=$(echo ${GDATE} | cut -c9-10)
 GDUMP=${GDUMP:-"gdas"}
+GDUMP_ENS=${GDUMP_ENS:-"enkf${GDUMP}"}
```

Comment:
This is a j-job. There is no one above this job setting `GDUMP_ENS`.
```suggestion
GDUMP_ENS="enkf${GDUMP}"
```
Same on the line above for `GDUMP`.